spec/rails_helper.rb

```diff
@@ -1,5 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
+require "view_component/test_helpers"
+require "capybara/rspec"
```

Comment:
Are these necessary? Tests pass for me without it, but maybe I'm missing something?

Reply:
They are unnecessary, was when we were trying to sort out what was going wrong with the templating lookup before the current implementation